client/components/notes-formatted-block/test/blocks.js

```diff
@@ -33,26 +26,34 @@ describe( 'Link block', () => {
 
 		const pathAbsoluteUrl = '/my/test/link?with=params&more=stuff+plus%20junk';
 		const text = 'my link text';
-		const link = shallow(
+
+		render(
 			<Blocks.Link
 				content={ { url: `https://wordpress.com${ pathAbsoluteUrl }` } }
 				children={ text }
 			/>
 		);
 
-		expect( link.prop( 'href' ) ).toEqual( pathAbsoluteUrl );
-		expect( link.text() ).toEqual( text );
+		const link = screen.getByRole( 'link' );
+
+		expect( link ).toHaveAttribute( 'href', pathAbsoluteUrl );
+		expect( link ).toHaveTextContent( text );
 	} );
 
 	test( 'on Jetpack Cloud, does not render links to WordPress.com', () => {
 		isJetpackCloud.mockImplementation( () => true );
 
 		const text = 'link text';
-		const link = shallow(
+
+		render(
 			<Blocks.Link content={ { url: 'https://wordpress.com/my/test/link' } } children={ text } />
 		);
 
-		expect( link ).toBeTextNodeWithValue( text );
+		const unlinkedText = screen.getByText( text );
+		const link = screen.queryByRole( 'link' );
+
+		expect( link ).not.toBeInTheDocument();
+		expect( unlinkedText ).toBeInTheDocument();
```

Comment:
I generally use `toBeVisible()` when testing visible items. I consider `.toBeVisible()` a more specific extension of `toBeInTheDocument()`, because something might be in the document, but be hidden for some reason.

This occurs a few more times in this PR, but this comment applies to them all.

Reply:
Yep, makes total sense. Thanks for the nudge about this!